src/tcgwars/logic/impl/gen4/MysteriousTreasures.groovy

```diff
@@ -157,9 +157,9 @@ public enum MysteriousTreasures implements LogicCardInfo {
   MULTI_ENERGY_118 ("Multi Energy", "118", Rarity.RARE, [SPECIAL_ENERGY, ENERGY]),
   DARKNESS_ENERGY_119 ("Darkness Energy", "119", Rarity.UNCOMMON, [SPECIAL_ENERGY, ENERGY]),
   METAL_ENERGY_120 ("Metal Energy", "120", Rarity.UNCOMMON, [SPECIAL_ENERGY, ENERGY]),
-  ELECTIVIRE_LV_X_121 ("Electivire LV.X", "121", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _LIGHTNING_]),
-  LUCARIO_LV_X_122 ("Lucario LV.X", "122", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _FIGHTING_]),
-  MAGMORTAR_LV_X_123 ("Magmortar LV.X", "123", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _FIRE_]),
+  ELECTIVIRE_LV_X_121 ("Electivire Lv.X", "121", Rarity.HOLORARE, [LVL_X, POKEMON, _LIGHTNING_]),
+  LUCARIO_LV_X_122 ("Lucario Lv.X", "122", Rarity.HOLORARE, [LVL_X, POKEMON, _FIGHTING_]),
+  MAGMORTAR_LV_X_123 ("Magmortar Lv.X", "123", Rarity.HOLORARE, [LVL_X, POKEMON, _FIRE_]),
```

Comment:
The implementations 👇 needs some ❤

Reply:
I'm planning to have the implementations be in a separate PR